src/NServiceBus.TransactionalSession/PipelineAwareSessionOptions.cs

```diff
@@ -0,0 +1,11 @@
+namespace NServiceBus.TransactionalSession
```

Comment:
This is a left over from some thought experiments. I originally thought it makes sense to float the invoke handler context into the session by calling Open and passing in those options. Then I kind of started debating whether it really makes sense because the session is implicitely open by the previous pipeline stage and I went with the current design.

That being said if we think calling the customization would be a better and more aligned approach I could call open and commit within the pipeline

You could argue most customizations need to pattern match anyway against the option type and should be save

https://github.com/Particular/NServiceBus.Persistence.AzureTable/blob/master/src/NServiceBus.Persistence.AzureTable.TransactionalSession/SetAsDispatchedHolderOpenSessionOptionCustomization.cs#L12-L15

Reply:
I pushed now a commit that shows the variant approach.

Reply:
Is this comment still relevant?